### DISCUSSION TITLE
Converting Tensorflow Model for use with ONNX

### DISCUSSION THREAD

**Discussion by helloimalastair** [webonnx/wonnx#187] [Q&A]
Hey y'all! I'm new to WONNX, and ML in general, so please bear with me if I am missing something obvious.

I have an Image Classifier model I've trained with Tensorflow, with the following architecture:
```py
from tensorflow import keras
from keras import layers
from keras.models import Sequential

data_augmentation = Sequential(
  [
    layers.RandomFlip("horizontal",
    input_shape=(img_height, img_width, 3)),
    layers.RandomRotation(0.1),
    layers.RandomZoom(0.1),
  ]
)

model = Sequential([
  data_augmentation,
  layers.Rescaling(1./255),
  layers.Conv2D(16, 3, padding='same', activation='relu'),
  layers.MaxPooling2D(),
  layers.Conv2D(32, 3, padding='same', activation='relu'),
  layers.MaxPooling2D(),
  layers.Conv2D(64, 3, padding='same', activation='relu'),
  layers.MaxPooling2D(),
  layers.Dropout(0.2),
  layers.Flatten(),
  layers.Dense(128, activation='relu'),
  layers.Dense(num_classes, name="outputs")
], "model_name")
```
This model works well against my test data, and I have converted it to an `ONNX` model with `tf2onnx.convert`. The trouble is when I try to import said model with `wonnx`, I get the error
```
opset error: the model references an unknown opset: 'ai.onnx.ml'
```
As far as I can tell, by default, `tf2onnx` will choose either the `ONNX_DOMAIN`(""), which should work with `wonnx`, or the `AI_ONNX_ML_DOMAIN`, and I'm not sure how to get it to compile with the first to make it work.

Anyone have any ideas what I could be missing?

**Comment by pixelspark** [comment]
You are not using any ops from ai.onnx.ml, so the quickest fix would be to remove the reference from the ONNX file after exporting it from TF (you could use the `ONNX` python package to do so, it is a bit cumbersome though)

**Reply by helloimalastair** [reply]
Do you know if there are any docs/posts about how to do so? I'm looking through the ONNX docs(specifically the [Python API](https://onnx.ai/onnx/repo-docs/PythonAPIOverview.html) and the [API Reference](https://onnx.ai/onnx/api/index.html)), but I'm not 100% sure what I'm looking for...
